### DISCUSSION TITLE
M1 MacOS overlays not showing?

### DISCUSSION THREAD

**Discussion by DaddyGoose24** [whs/runekit#47] [Q&A macOS]
Hello!

has anyone encountered this issue or a fix for it?

When I open runkit for clue solver is reading the screen and identifying the clues correctly. However, the overlays are missing on the solvers. When I tried runekit on my 10 year old Intel macbook air it was working correctly and displaying the overlays for the solver. I've made sure runekit has access to accessibility, recording and input.

**Comment by adamtha** [comment]
exact same here (still), couldnot make it work
